src/controllers/account.controller.ts

```diff
@@ -0,0 +1,73 @@
+import { AccountCreatedDetailsType } from '../constants/types';
+import {
+  getAllUserAccountsByUserIdService,
```

Comment:
You dont need to add service at the front of the name, its unnecessary. 
```suggestion
  getAllUserAccountsByUserId,
```

works

Reply:
i have removed all the service